src/date.rs

```diff
@@ -68,6 +68,9 @@ impl Date {
         Ok(Date { inner })
     }
 
+    // This is supposed to be `as_seconds_since_plist_epoch` if it takes a reference, or it can
+    // stay named the same and take a value. https://rust-lang.github.io/rust-clippy/master/index.html#wrong_self_convention
+    #[allow(clippy::wrong_self_convention)]
```

Comment:
@ebarnard this as a non-public api, should this be changed to `as_`, should it take a value instead of a ref or should it get an excemption?

Reply:
I think we want to mirror [`std::time::Duration`](https://doc.rust-lang.org/stable/std/time/struct.Duration.html) here, keep it taking a ref and rename to `as_`.

Reply:
Done.